Convert.py

```diff
@@ -1,36 +1,36 @@
+import os
 import sys
 import math
+import numpy as n
 import ROOT as r
-
-def benchmarkCorr(x,y,ecal, ecal_last, ehad, ehad_first):
-    a=1.0716
-    b=0.638
-    c=-0.0000174
-    ebench = ecal*a + ehad + b * math.sqrt(math.fabs(a*ecal_last*ehad_first)) + c(*ecal*a)**2
-
-def layerID(x,y):
-    R=math.sqrt(x**2+y**2)
-    if   R<1950: return 1
-    elif R<2000: return 2
-    elif R<2100: return 3
-    elif R<2200: return 4
-    elif R<2300: return 5
-    elif R<2400: return 6
-    elif R<2500: return 7
-    elif R<2540: return 8
-
-    elif R<2950: return 9
-    elif R<3100: return 10
-    elif R<3200: return 11
-    elif R<3400: return 12
-    elif R<3500: return 13
-    elif R<3600: return 14
-    elif R<3800: return 15
-    elif R<4100: return 16
-    elif R<4400: return 17
-    elif R<4600: return 18
-    else : return 99999
+from ROOT import gSystem
+gSystem.Load("libCaloAnalysis")
```

Comment:
to make this work we need this library available centrally?

Reply:
it is available, because I linked it to the local FCCSW installation. In the ini.sh there I defined the path to /afs/cern.ch/work/c/cneubuse/public/FCC_calo_analysis_cpp. SO it is automatically available as soon as you use the --local flag